kore/src/Kore/Parser/ParserImpl.hs

```diff
@@ -539,15 +542,9 @@ BNF definitions:
 koreVariableOrTermPatternParser :: Parser CommonKorePattern
 koreVariableOrTermPatternParser = do
     c <- ParserUtils.peekChar'
-    if c == '#'
-        then
-            asCommonKorePattern <$> variableOrTermPatternParser
-                korePatternParser
-                Meta
-        else
-            asCommonKorePattern <$> variableOrTermPatternParser
-                korePatternParser
-                Object
+    asCommonKorePattern <$> variableOrTermPatternParser
```

Comment:
Well, I wasn't sure we should do that anymore, since now it's just a wrapped variable, so whenever one removes the wrapper, it would be harder to distinguish between it and an element variable with the same name.